presto-blackhole/src/main/java/com/facebook/presto/plugin/blackhole/BlackHolePageSink.java

```diff
@@ -51,9 +51,9 @@ public BlackHolePageSink(ListeningScheduledExecutorService executorService, Dura
     }
 
     @Override
-    public Collection<Slice> finish()
+    public CompletableFuture<Collection<Slice>> finish()
     {
-        return ImmutableList.of();
+        return CompletableFuture.completedFuture(ImmutableList.of());
```

Comment:
Static import this method. (The name `completedFuture` is descriptive by itself, so the prefix `CompletableFuture.` just adds noise.)